app/controllers/LockableSettingsController.scala

```diff
@@ -55,11 +55,11 @@ class LockableSettingsController[T : Decoder : Encoder](
     * Returns current version of the settings in s3 as json, with the lock status.
     * The s3 data is validated against the model.
     */
-  def get= authAction.async {
+  def get= authAction.async { request =>
     withLockStatus { case VersionedS3Data(lockStatus, _) =>
       S3Json.getFromJson[T](dataBucket, dataKey)(s3Client).map {
         case Right(VersionedS3Data(value, version)) =>
-          Ok(S3Json.noNulls(LockableSettingsResponse(value, version, lockStatus).asJson))
+          Ok(S3Json.noNulls(LockableSettingsResponse(value, version, lockStatus, request.user.email).asJson))
```

Comment:
So now we can ensure that the user doesn't get the 'locked by user' message if they have the lock and refresh the page.